make build fails, exec: "stringer": executable file not found in $PATH

`make build` fails with the following error message:

```
$ make build
BIN_OUTPUT: bin/tabia
rm -rf bin/ cover.out
go generate ./...
go: downloading github.com/urfave/cli/v2 v2.2.0
go: downloading golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d
go: downloading github.com/shurcooL/githubv4 v0.0.0-20200414012201-bbc966b061dd
go: downloading github.com/antonmedv/expr v1.8.8
go: downloading golang.org/x/net v0.0.0-20200602114024-627f9648deb9
go: downloading github.com/shurcooL/graphql v0.0.0-20181231061246-d48a9a75455f
go: downloading github.com/cpuguy83/go-md2man/v2 v2.0.0
go: downloading github.com/russross/blackfriday/v2 v2.0.1
go: downloading github.com/shurcooL/sanitized_anchor_name v1.0.0
lib/github/github.go:32: running "stringer": exec: "stringer": executable file not found in $PATH
make: *** [generate] Error 1
```

--- a/README.md
+++ b/README.md
@@ -15,6 +15,19 @@ source .env
 env | grep TABIA
 ```
 
+Install required tools:
+
+```bash
+make install-tools
+```
+
+Add your $GO_PATH/bin folder to your path to make the installed Go tools globally available.
+
+```bash
+export GO_PATH="$HOME/go"
+export PATH="$GO_PATH/bin:$PATH"
+```
+
 ## Build
 
 To build the CLI you can make use of the `build` target using `make`.

